app/views/merchant/items/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1><%= @items[0].merchant.name %> Items</h1>
+<section class="grid-container">
+  <% @items.each do |item| %>
+    <section class = "grid-item" id= 'item-<%=item.id%>'>
+      <h2><%=link_to item.name, "/items/#{item.id}" %></h2>
+      <%=link_to image_tag(item.image), "/items/#{item.id}" %>
+      <p> <%= item.description unless @merchant%></p>
```

Comment:
If we're getting merchant name from the items or merchant, I remember mike saying that you should only need one variable, so in this case can we not grab the items from the merchant: @merchant.items?

Reply:
that was a good catch, adding as a refactor now and will push in next user story

Reply:
Yeah, we are already a merchant here so that line will always be truthy.